### PR TITLE
Update docker-images to v65

### PR DESCRIPTION
**Pull request by ebyhr**
## Description

Update docker-images to v65
The main change is Iceberg version up (`0.13.2` → `0.14.0`) https://github.com/trinodb/docker-images/pull/137

## Documentation

(x) No documentation is needed.

## Release notes

(x) No release notes entries required.

### CODE REVIEW

**Comment by ebyhr** [comment]
`TestIcebergSparkDropTableCompatibility.testCleanupOnDropTable` started failing.

```
2022-08-06T01:34:51.5411858Z tests               | 2022-08-06 07:19:51 INFO: FAILURE     /    io.trino.tests.product.iceberg.TestIcebergSparkDropTableCompatibility.testCleanupOnDropTable [TRINO, SPARK] (Groups: profile_specific_tests, iceberg) took 0.8 seconds
2022-08-06T01:34:51.5450406Z tests               | 2022-08-06 07:19:51 SEVERE: Failure cause:
2022-08-06T01:34:51.5451333Z tests               | org.junit.ComparisonFailure: [The data file /user/hive/warehouse/test_cleanup_on_drop_table1luqrck90l6b-e1488e7dcf7149d39b764c07b829fec7/data/20220806_013450_00052_iurj7-f0563caf-7565-4e40-9b48-3fe319e7dba8.orc removed after dropping the table] expected:<[fals]e> but was:<[tru]e>
2022-08-06T01:34:51.5452082Z tests               | 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
2022-08-06T01:34:51.5452870Z tests               | 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
2022-08-06T01:34:51.5453559Z tests               | 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
2022-08-06T01:34:51.5454151Z tests               | 	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
2022-08-06T01:34:51.5454852Z tests               | 	at io.trino.tests.product.iceberg.TestIcebergSparkDropTableCompatibility.assertFileExistence(TestIcebergSparkDropTableCompatibility.java:103)
2022-08-06T01:34:51.5455718Z tests               | 	at io.trino.tests.product.iceberg.TestIcebergSparkDropTableCompatibility.lambda$testCleanupOnDropTable$0(TestIcebergSparkDropTableCompatibility.java:86)
2022-08-06T01:34:51.5456327Z tests               | 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
2022-08-06T01:34:51.5456966Z tests               | 	at io.trino.tests.product.iceberg.TestIcebergSparkDropTableCompatibility.testCleanupOnDropTable(TestIcebergSparkDropTableCompatibility.java:86)
2022-08-06T01:34:51.5457656Z tests               | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2022-08-06T01:34:51.5458181Z tests               | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
2022-08-06T01:34:51.5458766Z tests               | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2022-08-06T01:34:51.5459380Z tests               | 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
2022-08-06T01:34:51.5459850Z tests               | 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
2022-08-06T01:34:51.5460322Z tests               | 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
2022-08-06T01:34:51.5460737Z tests               | 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
2022-08-06T01:34:51.5461159Z tests               | 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
2022-08-06T01:34:51.5461633Z tests               | 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
2022-08-06T01:34:51.5462112Z tests               | 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
2022-08-06T01:34:51.5462598Z tests               | 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
2022-08-06T01:34:51.5463100Z tests               | 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2022-08-06T01:34:51.5463513Z tests               | 	at java.base/java.lang.Thread.run(Thread.java:833)
2022-08-06T01:34:51.5463786Z tests               | 

2022-08-06T01:34:53.8292804Z tests               | 2022-08-06 07:19:53 INFO: FAILURE     /    io.trino.tests.product.iceberg.TestIcebergSparkDropTableCompatibility.testCleanupOnDropTable [SPARK, SPARK] (Groups: profile_specific_tests, iceberg) took 1.7 seconds
2022-08-06T01:34:53.8293797Z tests               | 2022-08-06 07:19:53 SEVERE: Failure cause:
2022-08-06T01:34:53.8294731Z tests               | org.junit.ComparisonFailure: [The data file /user/hive/warehouse/test_cleanup_on_drop_table1itsww49si3r/data/20220806_013452_00056_iurj7-ef29694f-f0a3-4373-aec4-d4e78a680ecb.parquet removed after dropping the table] expected:<[fals]e> but was:<[tru]e>
2022-08-06T01:34:53.8295557Z tests               | 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
2022-08-06T01:34:53.8296269Z tests               | 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
2022-08-06T01:34:53.8297163Z tests               | 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
2022-08-06T01:34:53.8297874Z tests               | 	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
2022-08-06T01:34:53.8299549Z tests               | 	at io.trino.tests.product.iceberg.TestIcebergSparkDropTableCompatibility.assertFileExistence(TestIcebergSparkDropTableCompatibility.java:103)
2022-08-06T01:34:53.8302282Z tests               | 	at io.trino.tests.product.iceberg.TestIcebergSparkDropTableCompatibility.lambda$testCleanupOnDropTable$0(TestIcebergSparkDropTableCompatibility.java:86)
2022-08-06T01:34:53.8303003Z tests               | 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
2022-08-06T01:34:53.8303768Z tests               | 	at io.trino.tests.product.iceberg.TestIcebergSparkDropTableCompatibility.testCleanupOnDropTable(TestIcebergSparkDropTableCompatibility.java:86)
2022-08-06T01:34:53.8304645Z tests               | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2022-08-06T01:34:53.8305286Z tests               | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
2022-08-06T01:34:53.8305878Z tests               | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2022-08-06T01:34:53.8306485Z tests               | 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
2022-08-06T01:34:53.8307074Z tests               | 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
2022-08-06T01:34:53.8307650Z tests               | 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
2022-08-06T01:34:53.8308073Z tests               | 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
2022-08-06T01:34:53.8308624Z tests               | 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
2022-08-06T01:34:53.8309223Z tests               | 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
2022-08-06T01:34:53.8309694Z tests               | 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
2022-08-06T01:34:53.8310293Z tests               | 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
2022-08-06T01:34:53.8310914Z tests               | 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
```

**Comment by ebyhr** [comment]
CI hit #13525

File: testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkDropTableCompatibility.java [hunk 1]
```diff
@@ -80,7 +80,12 @@ public void testCleanupOnDropTable(Engine tableCreatorEngine, Engine tableDroppe
         assertFileExistence(tableDirectory, true, "The table directory exists after creating the table");
         List<String> dataFilePaths = getDataFilePaths(tableName);
 
-        tableDropperEngine.queryExecutor().executeQuery("DROP TABLE " + tableName);
+        if (tableDropperEngine == SPARK) {
+            tableDropperEngine.queryExecutor().executeQuery("DROP TABLE " + tableName + " PURGE");
```
Comment (findepi) [inline testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkDropTableCompatibility.java:84]: Please add a comment why this change now.

**Review by findepi — APPROVED** [review]

**Comment by ebyhr** [comment]
Adjusted `assertFileExistence` argument because `DROP TABLE` with `PURGE` option caused hang-up as I left the code comment.
